.teamcity/settings.kts

```diff
@@ -769,6 +769,24 @@ object WpDesktop_DesktopE2ETests : BuildType({
 			dockerImage = "%docker_image_dekstop%"
 			dockerRunParameters = "-u %env.UID%"
 		}
+
+		script {
+			name = "Clean up artifacts"
+			executionMode = BuildStep.ExecutionMode.RUN_ON_SUCCESS
+			scriptContent = """
+				set -e
+				set -x
+
+				# Delete artifacts if branch is not trunk
+				if [ "%teamcity.build.branch.is_default%" != "true" ]; then
```

Comment:
Do you think we might as well discard all artifacts if it's a successful test run? (Referring to the logs and video/screenshots directories.)

Reply:
I can imagine cases where having screenshots/logs for green builds is helpful (for example, there could be some warning in the logs that I'm trying to address in my PR), and, compared to the compiled app, they are rather small.

I'd say keep them until they become a problem.